### PR TITLE
Added call to action, download page and fixed typos. 

### PR DESCRIPTION
**Pull request by avophile**

### CODE REVIEW

**Comment by mowchan** [comment]
I'd like to see a screenshot of the call to action.

**Comment by avophile** [comment]
I'm sorry I messed this one up so it cannot be automatically merged.

**Comment by avophile** [comment]
@mowchan 

call to action screenshot

**Comment by mowchan** [comment]
That button blends in. Maybe it could be the same color as links or that yellowish tone from the background image to give it a bit of pop.

**Comment by sugarPhlox** [comment]
Got a pic of the new button in action?

**Comment by avophile** [comment]
Sure!

inactive download lavender button

download lavender active

**Comment by sugarPhlox** [comment]
@noelseph , I want you to double check these colors!

**Comment by avophile** [comment]
Arguably, since they appear on the download page now, the download and installation sections should be eliminated from the Guide page (and navigation side bar and drop down!) in order to DRY. The Usage section could go either way.

File: css/main.css [hunk 1]
```diff
@@ -130,6 +130,7 @@ pre {
   white-space: -pre-wrap;
   white-space: -o-pre-wrap;
   word-wrap: break-word;
+
```
Comment (SamuelDavis) [inline css/main.css]: why whitespace?

File: css/main.css [hunk 2]
```diff
@@ -142,10 +143,10 @@ code {
 
 .btn-home { 
   color: #dad1d9; 
-  background-color: rgb(66,56,66);
-  background-color: rgba(66,56,66,.8);  
-  margin-top: 3px;
-  margin-bottom: 2px;
+  background: rgb(230,230,250);
+  background: rgba(230,230,250,0.4);
```
Comment (SamuelDavis) [inline css/main.css:146]: assigning value then overwriting it immediately?

File: views/guide.lavender [hunk 3]
```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```
Comment (SamuelDavis) [inline views/guide.lavender:223]: @mowchan is this space between if and elseif a convention we have/want to implement?

File: views/guide.lavender [hunk 4]
```diff
@@ -322,7 +325,7 @@ block content
             a(
               href='https://www.destroyallsoftware.com/talks/wat', 
               target='_blank') wat
```
Comment (SamuelDavis) [inline views/guide.lavender]: closing round bracket should always be on an unindented newline if all the attributes aren't on the same line

File: views/layouts/default.lavender [hunk 5]
```diff
@@ -83,4 +83,7 @@ body
                 a(href='/guide.html#langref') Language Reference
           li(class='#{active_class_about}')
             a(href='/about.html') About
+          li(class='#{active_class_download}')
```
Comment (SamuelDavis) [inline views/layouts/default.lavender]: you don't need to interpolate the active_class_download variable as a string. Within the parenthesis all variables are parsed normally. You can just write li(class=active_class_download)

File: views/guide.lavender (hunk 3 above)
Comment (avophile) [inline views/guide.lavender:223]: I added the space for readability. Probably not a good idea! I have not been able to get elseif to work at all (see Issue #53). I would be curious to know who has so I can figure out what I am doing wrong!

File: css/main.css (hunk 2 above)
Comment (mowchan) [inline css/main.css:146]: @SamuelDavis not all browsers support RGBA so you declare RGB as fallback.

File: views/layouts/default.lavender (hunk 5 above)
Comment (avophile) [inline views/layouts/default.lavender]: @SamuelDavis  This is good to know. I am trying to accomplish this a different way in my navbarsConquered branch.

**Comment by sugarPhlox** [comment]
@avophile , in reference to your earlier comment about removing the Installation from the language reference, I would say that is a grand idea to stop repetition :smile:

File: views/guide.lavender (hunk 3 above)
Comment (zlincoln) [inline views/guide.lavender:223]: jade syntax shows a space between 'else' and 'if', have you tried that?  maybe our lavender documentation is wrong.  @sugarPhlox can you confirm?

**Comment by zlincoln** [comment]
:cake: when whitespace issues are cleared up.  See comments in css and on your conditionals.

**Comment by sugarPhlox** [comment]
Yeah, I think I can cake as well once white space issues are fixed.

File: views/guide.lavender (hunk 3 above)
Comment (avophile) [inline views/guide.lavender:223]: @zlincoln I don't specifically remember trying that but feel like I tried every variation. 

I will experiment.

Comment (avophile) [inline views/guide.lavender:223]: @mowchan The spaces between if statements and else statements do not render. They just made the code more readable.

File: css/main.css (hunk 1 above)
Comment (sugarPhlox) [inline css/main.css]: @avophile , this white space still need to be removed.

**Comment by sugarPhlox** [comment]
:cake:

**Pull request closed without merging**